guides/Identities.md

```diff
@@ -0,0 +1,77 @@
+# About Identities
+
+Each request to a Wix API is associated with a specific identity type, which determines access permissions. If a [method has identity restrictions](#methods-with-restricted-identities), the method reference will include a note specifying the allowed identity types.
+Different contexts also limit the identities that can make a given call. For example, apps can only make calls as apps, not as visitors or members.
+
+Wix's identity types include:
+
+- [Site visitor](#site-visitor) 
+- [Site member](#site-member) 
+- [Wix user](#wix-user) 
+- [Wix app](#wix-app) 
+- [API key admin](#api-key-admin) 
+
+> **Note**: Additional identities are supported for app flows using the [SDK](https://dev.wix.com/docs/sdk).
+
+## Site visitor
+
+An anonymous site visitor who hasn't logged in. Methods that can be called by this identity usually involve operations specific to a particular visitor. These include accessing a list of products, creating and managing a cart, or accessing a login page to authenticate as a member.
```

Comment:
This first sentence definition is not a full sentence. So either write a full sentence or add a subheading such as Definition: An anonymous......

I think you can make each identity more structured.
Definition:
Context:
Example use case: